projects/frontegg-app/src/lib/frontegg-entitlements.service.ts

```diff
@@ -86,8 +26,14 @@ export class FronteggEntitlementsService {
    * @throws when entitlement is not enabled via frontegg options
    */
   public featureEntitlements$(feature: string, observer: PartialObserver<Entitlement>, customAttributes?: CustomAttributes): Subscription {
-    return this.getUserManipulatorSubscription<Entitlement>(
-      () => { return this.fronteggAppService.fronteggApp.getFeatureEntitlements(feature, customAttributes)},
+    return this.fronteggUserSubscriptionService.getUserManipulatorSubscription<Entitlement>(
+      (user: User) => { 
+        // the entitlemenets-common npm doesn't know to overcome the case of signed out user, then we get console errors 
```

Comment:
maybe better fixing `entitlemenets-common` package to avoid this error in other places?

Reply:
I agree but it won't happen now. I will ask them